ci/prod/dashboard.yaml

```diff
@@ -28,14 +28,150 @@ data:
       "links": [],
       "panels": [
         {
-          "cacheTimeout": null,
+          "alert": {
+            "alertRuleTags": {},
+            "conditions": [
+              {
+                "evaluator": {
+                  "params": [
+                    1500
+                  ],
+                  "type": "gt"
+                },
+                "operator": {
+                  "type": "and"
+                },
+                "query": {
+                  "params": [
+                    "A",
+                    "5m",
+                    "now"
+                  ]
+                },
+                "reducer": {
+                  "params": [],
+                  "type": "avg"
+                },
+                "type": "query"
+              }
+            ],
+            "executionErrorState": "alerting",
+            "for": "5m",
+            "frequency": "1m",
+            "handler": 1,
+            "message": "HSM looks sad",
```

Comment:
Arguably this could be a little more informative... But as it's for the team, perhaps that's not so urgent.